client/components/App/index.js

```diff
@@ -1,4 +1,5 @@
 import React, { PropTypes } from 'react'
+import 'foundation-apps/dist/css/foundation-apps.css'
```

Comment:
I found a different way around that issue, which we should talk about. I lean toward keeping like this though for now since it doesn't require anything special with loaders, since scss vs css is how we're deciding to treat something like our styles or vendor styles. Though, that's something that might need to be made more robust at some point.